File: src/main/java/org/manapart/jumpPads/ItemGroupJP.java
package org.manapart.jumpPads;

import net.minecraft.item.ItemGroup;
import net.minecraft.item.ItemStack;

public class ItemGroupJP extends ItemGroup {
    public static final ItemGroupJP instance = new ItemGroupJP(ItemGroup.getGroupCountSafe(), "jumppads");

    private ItemGroupJP(int index, String label) {
        super(index, label);
    }

    @Override
    public ItemStack makeIcon() {
        return new ItemStack(JumpPads.jumpPadsIcon);
    }

}

File: src/main/java/org/manapart/jumpPads/ItemGroupJP.kt
package org.manapart.jumpPads

import net.minecraft.world.item.CreativeModeTab
import net.minecraft.world.item.Item
import net.minecraft.world.item.ItemStack

class ItemGroupJP private constructor(index: Int, label: String) : CreativeModeTab(index, label) {
    override fun makeIcon(): ItemStack = ItemStack(JumpPads.icon)

    companion object {
        val instance = ItemGroupJP(getGroupCountSafe(), "jumppads")
    }
}